src/main/java/frc/robot/subsystems/Solver.java

```diff
@@ -0,0 +1,71 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.FeedbackDevice;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class Solver extends SubsystemBase {
+  // Motors
+  private static final TalonSRX solverMotor = new TalonSRX(Constants.CANIds.TalonSRX_Solver_Address); // Creates a new motor
+
+  /**
+   * Creates a new Solver.
+   */
+  public Solver() {
+    solverMotor.configFactoryDefault(); // Clear the programming to prevent weirdness
```

Comment:
what is weirdness?

Reply:
Motor controlers often tend to "glitch" upon sudden power shutdowns (power cycles) using the configFactoryDefault() protects against these.

Reply:
Have you tried turning it off and on again

Reply:
Yes thats why we use this line

Reply:
\:D